scripts/build.sh

```diff
@@ -64,7 +64,7 @@ fi
 [[ "$SNAPSHOT" == "true" ]] && VERSION=$VERSION-SNAPSHOT
 [ -z "$OUTPUT" ] && OUTPUT=artifacts
 
-./gradlew build -x test -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
+./gradlew assemble -x test -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
```

Comment:
Adding @gaiksaya, this is how we run the `assemble` task in default build script `./gradlew assemble --no-daemon --refresh-dependencies -DskipTests=true -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER`
https://github.com/opensearch-project/opensearch-build/blob/main/scripts/default/opensearch/build.sh#L73C55-L73C69

Reply:
Another example how security plugin runs https://github.com/opensearch-project/security/blob/main/scripts/build.sh#L71, please check

Reply:
I did a global search in opensearch-project for `-DskipTests` not sure where we define that property or if it comes with gradle. @owaiskazi19 Can you try to run the exact command on local (with snapshot is fine) just to make sure tests are skipped. If not we might need to add `-x integTest`

Reply:
> I did a global search in opensearch-project for `-DskipTests` not sure where we define that property 

That's actually a maven thing, not sure it has any effect here (the equivalent would be `-x test`)

Reply:
Tested`./gradlew assemble -x test` and tests didn't run.